meson.build

```diff
@@ -0,0 +1,17 @@
+project(
+  'flatpak-xdg-utils', 'c',
+  version: '0.0.1',
+  license: 'LGPL-2.1+',
+)
```

Comment:
```meson
default_options: [
  'buildtype=releasedebug',
  'warning_level=2',
]
```

Reply:
Thanks, added in the latest push to this branch.